analyzers/tests/SonarAnalyzer.UnitTest/Helpers/VbcHelperTest.cs

```diff
@@ -49,9 +45,7 @@ public class VbcHelperTest
         [DataRow("   eRrOr fOo:", true)]
         [DataRow("   error in foo:", false)]
         [DataRow("   error in foo :", false)]
-        public void IsTextMatchingVbcErrorPattern_ReturnsExpected(string text, bool expectedResult)
-        {
+        public void IsTextMatchingVbcErrorPattern_ReturnsExpected(string text, bool expectedResult) =>
```

Comment:
_No action needed:_ This is unrelated, it's better to limit scope of such big PRs